[AIRFLOW-252] Raise Sqlite exceptions when deleting tasks instance in WebUI

sekikn opened the pull request:
Dear Airflow Maintainers,

Please accept this PR that addresses the following issues:
- https://issues.apache.org/jira/browse/AIRFLOW-252

Testing Done:
- all 43 core unit tests passed
- confirmed delete button in "Task Instances" view worked with both SQLite and MySQL

If users who use SQLite as backend try to delete a task via browser, it fails with an exception.
Though this is a bug on Flask-Admin's side basically, this patch provides a workaround for it.

criccomini commented on airflow/www/views.py:
```diff
@@ -2236,6 +2237,17 @@ def set_task_instance_state(self, ids, target_state, session=None):
 
+    def get_one(self, id):
+        """
+            As a workaround for AIRFLOW-252, this method overrides Flask-Admin's ModelView.get_one().
```
Nit: no extra tab spacing needed here. Should align with """

sekikn committed:
[AIRFLOW-252] Raise Sqlite exceptions when deleting tasks instance in WebUI

If users who use SQLite as backend try to delete a task via browser, it fails with an exception.
Though this is a bug on Flask-Admin's side basically, this patch provides a workaround for it.
